### PR TITLE
Fix T292365 bug

### PR DESCRIPTION
**Pull request by helen-dikareva**
\cc @inikulin @churkin

### CODE REVIEW

**Comment by churkin** [comment]
lgtm

File: src/client/sandbox/event/focus-blur.js [hunk 1]
```diff
@@ -188,14 +188,12 @@ export default class FocusBlurSandbox extends SandboxBase {
 
             //if element was not focused and it has parent with tabindex, we focus this parent
             var parent             = el.parentNode;
-            var parentWithTabIndex = domUtils.closest(parent, '[tabindex]');
+            var parentWithTabIndex = parent !== document ? domUtils.closest(parent, '[tabindex]') : null;
```
Comment (inikulin) [inline src/client/sandbox/event/focus-blur.js]: Don't use reverse logic there possible, it's always harder to decipher:
```js
var parentWithTabIndex = parent === document ? null : domUtils.closest(parent, '[tabindex]');
```

**Comment by inikulin** [comment]
Regression test?

\r-

**Comment by helen-dikareva** [comment]
I fixed existing regression test for these conditions.
FPR

**Comment by inikulin** [comment]
@helen-dikareva What about comment in test?

**Comment by helen-dikareva** [comment]
Sorry. Added now.

**Comment by inikulin** [comment]
lgtm

**Pull request merged**